README.md

```diff
@@ -1,20 +1,41 @@
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-technocrats/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-technocrats?branch=develop)
```

Comment:
This is unrelated to the deploy task.

Reply:
Thanks for your review @Mnickii. I have resolved the issue.